package.nls.json

```diff
@@ -0,0 +1,205 @@
+	"githubPullRequests.queries.label.description": "The label to display for the query in the Pull Requests tree",
+	"githubPullRequests.queries.query.description": "The query used for searching pull requests.",
+	"githubPullRequests.defaultMergeMethod.description": "The method to use when merging pull requests.",
+	"githubPullRequests.notifications.description": "If GitHub notifications should be shown to the user.",
+	"githubPullRequests.fileListLayout.description": "The layout to use when displaying changed files list.",
+	"githubPullRequests.defaultDeletionMethod.selectLocalBranch.description": "When true, the option to delete the local branch will be selected by default when deleting a branch from a pull request.",
+	"githubPullRequests.defaultDeletionMethod.selectRemote.description": "When true, the option to delete the remote will be selected by default when deleting a branch from a pull request.",
+	"githubPullRequests.terminalLinksHandler.description": "Default handler for terminal links.",
+	"githubPullRequests.terminalLinksHandler.github": "Create the pull request on GitHub",
+	"githubPullRequests.terminalLinksHandler.vscode": "Create the pull request in VS Code",
+	"githubPullRequests.terminalLinksHandler.ask": "Ask which method to use",
+	"githubPullRequests.createOnPublishBranch.description": "Create a pull request when a branch is published.",
+	"githubPullRequests.createOnPublishBranch.never": "Never create a pull request when a branch is published.",
+	"githubPullRequests.createOnPublishBranch.ask": "Ask if you want to create a pull request when a branch is published.",
+	"githubPullRequests.commentExpandState.description": "Controls whether comments are expanded when a document with comments is opened.",
+	"githubPullRequests.commentExpandState.expandUnresolved": "All unresolved comments will be expanded.",
+	"githubPullRequests.commentExpandState.collapseAll": "All comments will be collapsed",
+	"githubPullRequests.useReviewMode.description": "Choose which pull request states will use review mode. \"Open\" pull requests will always use review mode.",
+	"githubPullRequests.useReviewMode.merged": "Use review mode for merged pull requests.",
+	"githubPullRequests.useReviewMode.closed": "Use review mode for closed pull requests. Merged pull requests are not considered \"closed\".",
+	"githubPullRequests.assignCreated.description": {
+		"message": "All pull requests created with this extension will be assigned to this user. To assign to yourself, use the '${user}' variable.",
+		"comment": [
+			"{Locked=']${user}'}",

```

Comment:
```suggestion
			"{Locked='${user}'}",

```

Reply:
I mislead you hear I think... the `]` in the example was to prevent translators from putting a space between`]` and `(` in the markdown syntax `[]()`